app/policies/external_activity_policy.rb

```diff
@@ -30,7 +30,7 @@ def set_private_before_matedit?
   end
 
   def copy?
-    user.present?
+    not_anonymous?
```

Comment:
So the better fix would be to change this line of the spec: https://github.com/concord-consortium/rigse/blob/master/spec/policies/external_activity_policy_spec.rb#L5 to be `let(:active_user)       { nil          }`  
That might break other specs, which we should then fix.